src/tm_data_types/files_and_formats/wfm/data_formats/digital.py

```diff
@@ -19,7 +19,7 @@ class WaveformFileWFMDigital(WFMFile[DigitalWaveform]):
     ################################################################################################
 
     _META_DATA_LOOKUP = WFMFile.update_bidict(
-        WFMFile._META_DATA_LOOKUP,
+        WFMFile._META_DATA_LOOKUP,  # noqa: SLF001
         {
             "digital_probe_0_state": "d0",
```

Comment:
Noe: add issue to figure out what d0-d7 even mean. The assumption is they represent whether the digital probe was turned off or on, but I remember that not being the case, so this name may be non representative.

Reply:
@qthompso, please go ahead and file an issue